src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx

```diff
@@ -58,7 +67,7 @@ export const DataFileButton: FunctionComponent<DataFileButton> = ({ onDataFile,
   };
 
   // eslint-disable-next-line @typescript-eslint/no-shadow
-  const validateDataFile = (schema: Schema, data: unknown): ValidateDataFile => {
+  const validateDataFile = (schema: AnySchema, data: unknown): ValidateDataFile => {
```

Comment:
this schema type here cannot use `JSONSchema`?

Reply:
ok will update to use JSONSchema